lib/ts/recipe/session/recipe.ts

```diff
@@ -28,9 +28,9 @@ import RecipeImplementationWithJWT from "./with-jwt";
 import { Querier } from "../../querier";
 import APIImplementation from "./api/implementation";
 import { BaseRequest, BaseResponse } from "../../framework";
-import JWTRecipe from "../jwt/recipe";
 import OverrideableBuilder from "supertokens-js-override";
 import { APIOptions } from ".";
+import OpenIDRecipe from "../openid/recipe";
```

Comment:
Why using "ID" in some places and "Id" in others? Stick to one.

Reply:
Done, using OpenId everywhere